### PR TITLE
Path error message

### PR DESCRIPTION
**Pull request by dannasman**
Hi!

Here is a solution proposal to #4988 (error message part).

### CODE REVIEW

**Comment by dannasman** [comment]
To my understanding the failing MacOS tests do not seem to be related to the changes. Not quite sure though.

**Comment by youknowone** [comment]
Thank you for contributing! The changes looks good. Let me look more in other test issues.

**Review by youknowone — APPROVED** [review]
Looking good, thank you so much!

**Comment by youknowone** [comment]
And I am really sorry to be late! I was heavily occupied for last a few weeks 😂

**Comment by dannasman** [comment]
Great! Understandable, no need to be sorry👍

File: vm/src/vm/mod.rs [hunk 1]
```diff
@@ -223,12 +223,19 @@ impl VirtualMachine {
     #[cfg(feature = "encodings")]
     fn import_encodings(&mut self) -> PyResult<()> {
         self.import("encodings", None, 0).map_err(|import_err| {
-            let err = self.new_runtime_error(
-                "Could not import encodings. Is your RUSTPYTHONPATH set? If you don't have \
+            let msg = if !self.state.settings.path_list.iter().any(|s| s == "PYTHONPATH" || s == "RUSTPYTHONPATH"){
```
Comment (youknowone) [inline vm/src/vm/mod.rs:226]: ah, this is not correct. `path_list` only have path. `PYTHONPATH` is an env name. It never matches.

**Pull request merged**